### PR TITLE
add default_server to sed file for rhscl as well

### PR DESCRIPTION
**Pull request by pkubatrh**

### CODE REVIEW

File: 1.18/root/opt/app-root/nginxconf.sed [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/listen/s%80%8080%
+/listen/s%80%8080 default_server%
```
Comment (phracek) [inline 1.18/root/opt/app-root/nginxconf.sed:1]: Only one question. This is relevant only for version `1.18`? What about the rest?

**Review by phracek — CHANGES_REQUESTED** [review]

File: 1.18/root/opt/app-root/nginxconf.sed (hunk 1 above)
Comment (phracek) [reply 1.18/root/opt/app-root/nginxconf.sed:1]: It is relevant only for 1.18. Then LGTM.

**Review by phracek — APPROVED** [review]
Thanks for the fix. LGTM.

**Comment by phracek** [comment]
[test]

**Comment by pkubatrh** [comment]
The rhel8 failure seems like our known podman issue with CID files, so lest merge. Thanks for review Petr!

**Pull request merged**